src/nspawn/nspawn-register.c

```diff
@@ -132,7 +132,11 @@ int register_machine(
                                            * container to ever create
                                            * these device nodes. */
                                           "/dev/pts/ptmx", "rw",
-                                          "char-pts", "rw");
+                                          "char-pts", "rw",
+                                          /* Allow /run/systemd/inaccessible/{chr,blk}
+                                           * devices inside the container */
+                                          "/run/systemd/inaccessible/chr", "rwm",
+                                          "/run/systemd/inaccessible/blk", "rwm");
```

Comment:
Does it ignore the error if the path does not exist? If not, is it possible to add the magic "ignore" character, like "-/run/systemd/inaccessible/chr"?

My concern is with rkt: if we have systemd-v229 on the host, those "chr" and "blk" files will not exist on the host. Then, if rkt uses systemd-nspawn-v231, it will send the message to systemd-machined-v229 & systemd-v229, and systemd on the host will fail.

Ideally, we would like fallback to read-only: that's better than read-write, but it might be too difficult to do. Better ideas welcome :)

/cc @lucab